utils/oomparser/testmessagefeed.txt

```diff
@@ -0,0 +1,69 @@
+Jan 21 22:01:49 localhost kernel: [62278.816267] ruby invoked oom-killer: gfp_mask=0x201da, order=0, oom_score_adj=0
```

Comment:
Lets add some bogus (real looking, but unrelated) lines before and after the OOM.